internal/venice-common/src/test/resources/testHybridStore.json

```diff
@@ -0,0 +1,266 @@
+  "owner": "urn:li:corpuser:testuser",
+  "createdTime": 1661227581365,
+  "persistenceType": "ROCKS_DB",
+  "routingStrategy": "CONSISTENT_HASH",
+  "readStrategy": "ANY_OF_ONLINE",
+  "offLinePushStrategy": "WAIT_N_MINUS_ONE_REPLCIA_PER_PARTITION",
+  "currentVersion": 818,
+  "storageQuotaInByte": 53687091200,
+  "readQuotaInCU": 5000,
+  "hybridStoreConfig": {
+    "rewindTimeInSeconds": 86400,
+    "offsetLagThresholdToGoOnline": 10,
+    "producerTimestampLagThresholdToGoOnlineInSeconds": -1,
+    "dataReplicationPolicy": "NON_AGGREGATE",
+    "bufferReplayPolicy": "REWIND_FROM_EOP"
+  },
+  "partitionerConfig": {
+    "partitionerClass": "com.linkedin.venice.partitioner.DefaultVenicePartitioner",
+    "partitionerParams": {},
+    "amplificationFactor": 1
+  },
+  "replicationFactor": 3,
+  "lowWatermark": 0,
+  "partitionCount": 0,
+  "daVinciPushStatusStoreEnabled": true,
+  "hybrid": false,
+  "nativeReplicationSourceFabric": "prod-lva1",
+  "rmdChunkingEnabled": false,
+  "accessControlled": true,
+  "rmdVersion": -1,
+  "schemaAutoRegisterFromPushJobEnabled": false,
+  "readComputationEnabled": false,
+  "latestSuperSetValueSchemaId": -1,
+  "hybridStoreDiskQuotaEnabled": false,
+  "storeMetadataSystemStoreEnabled": false,
+  "latestVersionPromoteToCurrentTimestamp": 1733110839107,
+  "migrationDuplicateStore": false,
+  "systemStores": {
+    "venice_system_store_davinci_push_status_store": {
+      "currentVersion": 3,
+      "latestVersionPromoteToCurrentTimestamp": 1729632343583,
+      "versions": [
+        {
+          "storeName": "testStore",
+          "number": 3,
+          "createdTime": 1729632323698,
+          "pushJobId": "SYSTEM_STORE_PUSH_1729632306180",
+          "partitionCount": 1,
+          "partitionerConfig": {
+            "partitionerClass": "com.linkedin.venice.partitioner.DefaultVenicePartitioner",
+            "partitionerParams": {},
+            "amplificationFactor": 1
+          },
+          "replicationFactor": 3,
+          "hybridStoreConfig": {
+            "rewindTimeInSeconds": 86400,
+            "offsetLagThresholdToGoOnline": 10,
+            "producerTimestampLagThresholdToGoOnlineInSeconds": -1,
+            "dataReplicationPolicy": "NON_AGGREGATE",
+            "bufferReplayPolicy": "REWIND_FROM_EOP"
+          },
+          "nativeReplicationSourceFabric": "prod-lva1",
+          "rmdChunkingEnabled": false,
+          "blobTransferEnabled": false,
+          "versionSwapDeferred": false,
+          "minActiveReplicas": 2,
+          "useVersionLevelIncrementalPushEnabled": true,
+          "useVersionLevelHybridConfig": true,
+          "dataRecoveryVersionConfig": null,
+          "repushSourceVersion": -1,
+          "chunkingEnabled": true,
+          "incrementalPushEnabled": false,
+          "separateRealTimeTopicEnabled": false,
+          "nativeReplicationEnabled": true,
+          "pushStreamSourceAddress": "kafka.venice.kafka.prod-lva1.atd.prod.linkedin.com:16637",
```

Comment:
Thanks for adding the test! However, I don't think we can expose LI specific urls to open source.